### PR TITLE
fix some typo on doc

### PR DESCRIPTION
**Pull request by tgfree7**
Signed-off-by: tgfree <tgfree7@gmail.com>

### CODE REVIEW

File: charts/metallb/README.md [hunk 1]
```diff
@@ -12,108 +12,108 @@ A network load-balancer implementation for Kubernetes using standard routing pro
...
-| speaker.readinessProbe.successThreshold | int | `1` |  |
-| speaker.readinessProbe.timeoutSeconds | int | `1` |  |
-| speaker.resources | object | `{}` |  |
-| speaker.runtimeClassName | string | `""` |  |
-| speaker.serviceAccount.annotations | object | `{}` |  |
-| speaker.serviceAccount.create | bool | `true` |  |
-| speaker.serviceAccount.name | string | `""` |  |
-| speaker.tolerateMaster | bool | `true` |  |
-| speaker.tolerations | list | `[]` |  |
+| Key                                                                      | Type   | Default                        | Description                                                                             |
```
Comment (fedepaol) [inline charts/metallb/README.md:15]: This is autogenerated and we have CI enforcing, so I'd avoid changing or CI will never pass

File: design/0001-frr.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ provides detailed logging features to aid debugging.
 * Provide a method to integrate arbitrary BGP implementations into MetalLB.
 * Support runtime changing of BGP implementation type.
 * Extend the functionality of the MetalLB BGP implementation in in order to
-  take advantage of the additional cababilities of FRR.
+  take advantage of the additional capabilities of FRR.
```
Comment (fedepaol) [inline design/0001-frr.md:49]: Despite these being typos, I don't want to change the design proposals after merging them to keep the original state.
It's not public facing doc so I think it's fine.

File: design/pool-configuration.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ into the CRDs, or provide some offline tool in charge of that (the latter option
 
 #### Impact on users currently using the MetalLB operator
 
-The operator already exposes CRDs, mapped directly to the current configmap entitites.
+The operator already exposes CRDs, mapped directly to the current configmap entities.
```
Comment (fedepaol) [inline design/pool-configuration.md:141]: same

File: website/content/installation/_index.md [hunk 4]
```diff
@@ -164,13 +164,13 @@ and change the `BGP_TYPE` environment variable of the `manager` container to `fr
...
 The FRR daemons logging level are configured using the speaker `--log-level` argument following the below mapping:
 
-Speaker log level | FRR log level
-------------------|--------------
-all, debug        | debugging
-info              | informational
-warn              | warnings
-error             | error
-none              | emergencies
+| Speaker log level | FRR log level |
```
Comment (fedepaol) [inline website/content/installation/_index.md:167]: What's the reason for changing this?

**Comment by fedepaol** [comment]
Hi, thanks for the PR, left a few comments. Also, would you mind changing the commit subject / body? Per contributors guidelines, we are trying to stick to https://chris.beams.io/posts/git-commit/

**Comment by tgfree7** [comment]
> Hi, thanks for the PR, left a few comments. Also, would you mind changing the commit subject / body? Per contributors guidelines, we are trying to stick to https://chris.beams.io/posts/git-commit/

ok

**Pull request merged**